From 80c921ffdd494d0a623b1fb829528ae18635645b Mon Sep 17 00:00:00 2001
From: Scott Koranda <scott.koranda@ligo.org>
Date: Fri, 13 Nov 2009 05:47:52 -0600
Subject: [PATCH] Removed import of popen2 module, deprecated

The use of the popen2 module is deprecated and is
no longer used in the ligo_data_find client tool.
---
 glue/bin/ligo_data_find |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/glue/bin/ligo_data_find b/glue/bin/ligo_data_find
index 496d6ff..56401e1 100644
--- a/glue/bin/ligo_data_find
+++ b/glue/bin/ligo_data_find
@@ -28,7 +28,6 @@ import sys
 import os
 import getopt
 import re
-import popen2
 import exceptions
 import httplib
 import urlparse
-- 
1.6.5.2

